docs/apigateway.md

```diff
@@ -8,6 +8,11 @@ This experimental feature will allow you to invoke an action with HTTP methods o
 
 Use the CLI to expose your OpenWhisk actions through the OpenWhisk API Gateway. 
 
+## OpenWhisk CLI configuration for api-experimental command
```

Comment:
i think we can remove the "for..." in the section title like so:
-> ## OpenWhisk CLI configuration